concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-362	362	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Schwester der eigenen Frau	sister of one's wife		
+Steinthal-1875-1565-363	363	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Schwester des eigenen Mannes	sister of one's husband		
+Steinthal-1875-1565-364	364	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Bruder der eigenen Frau	brother of one's wife	2380	BROTHER-IN-LAW (OF MAN)
+Steinthal-1875-1565-365	365	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Bruder des eigenen Mannes	brother of one's husband		
+Steinthal-1875-1565-366	366	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Mann der eigenen Schwester	husband of one's sister		
+Steinthal-1875-1565-367	367	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Frau des eigenen Bruders	wife of one's brother		
+Steinthal-1875-1565-368	368	Greis	Greis	old man	406	OLD MAN
+Steinthal-1875-1565-369	369	alte Frau	alte Frau	old woman	907	OLD WOMAN
+Steinthal-1875-1565-370	370	Er ist alt.	Er ist alt.	He is old.		
+Steinthal-1875-1565-371	371	Er altert.	Er altert.	He is ageing.		
+Steinthal-1875-1565-372	372	Er bekommt graue Haare.	Er bekommt graue Haare.	He is greying.		
+Steinthal-1875-1565-373	373	Er ist gestorben, todt, alt geworden, nicht alt geworden, noch jung.	Er ist gestorben.	He died.		
+Steinthal-1875-1565-374	374	Er ist gestorben, todt, alt geworden, nicht alt geworden, noch jung.	Er ist todt.	He is dead.		
+Steinthal-1875-1565-375	375	Er ist gestorben, todt, alt geworden, nicht alt geworden, noch jung.	Er ist alt geworden.	He grew old.		
+Steinthal-1875-1565-376	376	Er ist gestorben, todt, alt geworden, nicht alt geworden, noch jung.	Er ist nicht alt geworden.	He died at a young age.		
+Steinthal-1875-1565-377	377	Er ist gestorben, todt, alt geworden, nicht alt geworden, noch jung.	Er ist noch jung.	He is still young.		
```

Comment:
Maybe 1207 YOUNG?

Reply:
I would rather leave it unmapped because I am unsure about the connotations with "still".